modules/ROOT/pages/6.1.2-release-notes.adoc

```diff
@@ -0,0 +1,95 @@
+= TinyMCE 6.1.2
+:navtitle: TinyMCE 6.1.2
+:description: Release notes for TinyMCE 6.1.2
+:keywords: releasenotes, new, changes, bugfixes
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.1.2 was released for {enterpriseversion} and {cloudname} on Wednesday, August 3^rd^, 2022. It includes {productname} 6.1.2. These release notes provide an overview of the changes for {productname} 6.1.2:
+
+* xref:bug-fixes[Bug fixes]
+* xref:known-issues[Known issues]
+
+[[bug-fixes]]
+== Bug fixes
+
+
+=== Invalid special elements were not cleaned up correctly during sanitization
+
+In {productname} 5.x, loading data that contained invalid elements resulted in the invalid elements and their contents being removed before the data was placed in a {productname} document.
```

Comment:
```suggestion
In {productname} 5.x, loading data that contained invalid special elements resulted in the invalid elements and their contents being removed before the data was placed in a {productname} document.
```
This only happens for special elements, not all invalid elements. For all others they are just unwrapped instead.